README.md

```diff
@@ -43,6 +41,9 @@ PcapPlusPlus enables decoding and forging capabilities for a large variety of ne
 
 You can choose between downloading from GitHub release page, use a package manager or build PcapPlusPlus yourself. For more details please visit the [Download](https://pcapplusplus.github.io/docs/install) page in PcapPlusPlus web-site.
 
+![GitHub all releases](https://img.shields.io/github/downloads/seladb/PcapPlusPlus/total?style=flat&label=Downloads&logo=github)
+![homebrew downloads](https://img.shields.io/homebrew/installs/dy/pcapplusplus?style=flat&label=Downloads&logo=homebrew)

```

Comment:
homebrew is just one of the package managers that host PcapPlusPlus so this number doesn't reflect the real downloads. Besides 500/year seems low... maybe we should avoid showing it?

Reply:
Unfortunately, other platforms do not provide this functionality (or at least shields.io not support it). So, removing homebrew will be a better choice